### PR TITLE
In 'download' command filter files that are not related to plugin 

### PR DESCRIPTION
**Pull request by frombetelgeuse**
- auto-reload project at the end

### CODE REVIEW

File: src/main/java/com/crowdin/action/DownloadAction.java [hunk 1]
```diff
@@ -45,11 +77,31 @@ private void extractTranslations(File archive) {
...
-            zipFile.extractAll(archive.getParent());
-            NotificationUtil.showInformationMessage("Translations successfully downloaded");
+            zipFile.extractAll(dirPath);
         } catch (ZipException e) {
             NotificationUtil.showInformationMessage("Downloading translations failed");
             e.printStackTrace();
         }
     }
+
+    private List<File> walkDir(Path dir) {
```
Comment (yevheniyJ) [inline src/main/java/com/crowdin/action/DownloadAction.java:87]: Move to FileUtil class

File: src/main/java/com/crowdin/action/DownloadAction.java [hunk 2]
```diff
@@ -45,11 +77,31 @@ private void extractTranslations(File archive) {
...
+                .filter(java.nio.file.Files::isRegularFile)
+                .map(Path::toFile)
+                .collect(Collectors.toList());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private List<String> filterFiles(List<String> files, List<String> androidCodes) {
+        Pattern pattern = Pattern.compile("[\\\\\\\\\\/]?values-([a-zA-Z-]+)[\\\\\\\\\\/]([^\\\\\\\\|^\\/]+)$");
```
Comment (yevheniyJ) [inline src/main/java/com/crowdin/action/DownloadAction.java:99]: Create a static final field. No need to compile pattern each time when this method is invoked

File: src/main/java/com/crowdin/action/DownloadAction.java (hunk 1 above)
Comment (frombetelgeuse) [reply src/main/java/com/crowdin/action/DownloadAction.java]: Done

File: src/main/java/com/crowdin/action/DownloadAction.java (hunk 2 above)
Comment (frombetelgeuse) [reply src/main/java/com/crowdin/action/DownloadAction.java]: Fixed

**Pull request merged**